### PR TITLE
tkt-72113: New API endpoint for Boot Scrub

### PR DESCRIPTION
**Pull request by sonicaj**
This commit adds a new endpoint for updating boot scrub interval value as the previous case made us validate all the other system.advanced service attributes as well.

### CODE REVIEW

File: src/middlewared/middlewared/plugins/boot.py [hunk 1]
```diff
@@ -203,3 +204,25 @@ class BootService(Service):
         """
         subjob = await self.middleware.call('zfs.pool.scrub', 'freenas-boot')
         return await job.wrap(subjob)
+
+    @accepts(
+        Int('interval', validators=[Range(min=1)])
+    )
+    async def scrub_interval(self, interval):
```
Comment (william-gr) [inline src/middlewared/middlewared/plugins/boot.py]: If one is get_scrub_interval it only makes sense this one is called set_scrub_interval

Comment (sonicaj) [reply src/middlewared/middlewared/plugins/boot.py]: agreed

**Comment by sonicaj** [comment]
Thank you :)

**Comment by sonicaj** [comment]
backport

**Pull request merged**